### PR TITLE
add parameters for Fusion

### PR DESCRIPTION
**Pull request by tomaszwroblewski**
Extended suppor for Fusion ad server.

### CODE REVIEW

**Comment by tomaszwroblewski** [comment]
I have added support for extra parameters. Collection should be added as key-value in query string form.

**Comment by tomaszwroblewski** [comment]
I know there is a problem with CLS status in our case. We signed that already but system can't recognise this for some reason. We tried to solve this while working on last pull request(https://github.com/ampproject/amphtml/pull/6148) without any luck.

File: ads/fusion.js [hunk 1]
```diff
@@ -16,25 +16,36 @@
 
 import {writeScript, validateData} from '../3p/3p';
 
+function queryParametersToObject(input) {
+  if (!input) { return undefined; }
```
Comment (zhouyx) [inline ads/fusion.js]: function in new line please.

File: ads/fusion.js [hunk 2]
```diff
@@ -16,25 +16,36 @@
...
+}
+
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function fusion(global, data) {
-  validateData(data, [], ['mediaZone', 'layout', 'adServer', 'space']);
+  validateData(data, [],
+    ['mediaZone', 'layout', 'adServer', 'space', 'parameters']);
```
Comment (zhouyx) [inline ads/fusion.js]: extra indentation please

File: ads/fusion.js [hunk 3]
```diff
@@ -16,25 +16,36 @@
...
   const container = global.document.getElementById('c');
   const ad = global.document.createElement('div');
   ad.setAttribute('data-fusion-space', data.space);
   container.appendChild(ad);
+  const parameters = queryParametersToObject(data.parameters);
 
-  writeScript(global, 'https://assets.adtomafusion.net/fusion/latest/fusion-amp.min.js', () => {
-    global.Fusion.apply(container, global.Fusion.loadAds(data));
+  writeScript(global,
+    'https://assets.adtomafusion.net/fusion/latest/fusion-amp.min.js', () => {
```
Comment (zhouyx) [inline ads/fusion.js]: extra indentation please

**Comment by tomaszwroblewski** [comment]
@zhouyx Thanks, I have applied your suggestions.

**Comment by zhouyx** [comment]
Ping @erwinmombay to merge.

**Pull request merged**